.github/workflows/ci.yml

```diff
@@ -148,3 +149,11 @@ jobs:
           docker pull --platform linux/amd64 denoland/deno:latest
           docker pull --platform linux/arm64 denoland/deno:${{ github.ref_name }}
           docker pull --platform linux/arm64 denoland/deno:latest
+
+      - name: Update Docker Hub Description
+        uses: peter-evans/dockerhub-description@e98e4d1628a5f3be2be7c231e50981aee98723ae # v4.0.0
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+          repository: denoland/deno
+          short-description: ${{ github.event.repository.description }}
```

Comment:
I think maybe we should change this to only happen on a tag?

Reply:
Actually, probably just copying the above so this only happens for one of the runs too:

```
if: github.repository == 'denoland/deno_docker' && github.ref_type == 'tag' && matrix.kind == 'debian'
```

Reply:
Good catch!